### PR TITLE
LPS-180927 Automate CP2ProjectSelection#AssertThatAbleToSearchWithProjectCode

### PR DESCRIPTION
**Pull request by styller1**
Ticket:
[LPS-180297](https://liferay.atlassian.net/browse/LPS-180927)

### CODE REVIEW

**Comment by styller1** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase:1]: Hi, Pablo! Can you change the commit message to "LPS-180927 Add AssertThatAbleToSearchWithProjectCode test"? I think it is clearer this way. What you think?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase [hunk 1]
```diff
@@ -53,6 +53,26 @@ definition {
 			value = "Test Test");
 	}
 
+	@description = "Story: LPS-167803 | Test Summary: Verify that it is possible to search for a project with just the project code"
+	@priority = 2
+	test AssertThatAbleToSearchWithProjectCode {
+		task ("Given the User go to Customer Portal Site") {
```
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase:59]: Change task description to "Given that the User goes to Customer Portal Site"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase [hunk 2]
```diff
@@ -53,6 +53,26 @@ definition {
...
+			ApplicationsMenu.gotoSite(key_site = ${testSiteName});
+		}
+
+		task ("When search for a project by the ERC") {
```
Comment (heiditrindade) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase:63]: "When the User searches for a project by ERC"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase
Comment (styller1) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase:1]: I agree

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase (hunk 1 above)
Comment (styller1) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase:59]: Okay

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase (hunk 2 above)
Comment (styller1) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/tests/CustomerPortalSite.testcase:63]: Okay

**Comment by heiditrindade** [comment]
ci:forward

**Pull request closed without merging**